lib/rewriteUrl.js

```diff
@@ -1,6 +1,12 @@
 function interpolateIdIfAvailable(url) {
```

Comment:
Is there any reason this can't use url-parse?


```
var urlParse = require('url-parse')
const url = urlParse('http://www.nhs.uk/Services/gp/Overview/DefaultView.aspx?id=35956', true);
console.log(url.query.id);
```
[RunKit](https://runkit.com/58d8f244adc84900149411df/58d8f244adc84900149411e0)